src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -49,12 +50,19 @@
     protected final Permission PCREATEBI = new Permission("tradeshop.create.bi");
     
     protected final TradeShop plugin = (TradeShop) Bukkit.getPluginManager().getPlugin("TradeShop");
-	
+
+    protected final UUID KOPUUID = UUID.fromString("daf79be7-bc1d-47d3-9896-f97b8d4cea7d");
```

Comment:
don't want those to be public as there will be a getter method, they are there so that i can have it tell us the version being run on the server when we log in